techeria_controller/techeria_app/models.py

```diff
@@ -20,16 +20,16 @@ class Meta:
 class BuyerModel(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
-    user_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
     date_of_birth = models.DateField()
     email = models.EmailField()
-    mobile_number = models.IntegerField(max_length=10)
+    mobile_number = models.IntegerField()
```

Comment:
we can use the phonenumber_field library as we did for email. what do you think

Reply:
Got it Farah. Thanks for your participation